kvirt/providers/gcp/__init__.py

```diff
@@ -276,11 +276,17 @@ def create(self, name, virttype=None, profile='', flavor=None, plan='kvirt', cpu
                     disksize = 20
                     pprint("Rounding primary disk to to 20Gb")
                 newdisk['initializeParams'] = {'sourceImage': src, 'diskSizeGb': disksize}
+                boot_disk_type = overrides.get('boot_disk_type', None)
+                if boot_disk_type:
+                    newdisk['initializeParams']['diskType'] = f'/compute/v1/projects/{project}/zones/{zone}/diskTypes/{boot_disk_type}'
                 newdisk['boot'] = True
             else:
                 diskname = f"{name}-disk{index}"
                 diskpath = f'/compute/v1/projects/{project}/zones/{zone}/disks/{diskname}'
                 info = {'name': diskname, 'sizeGb': disksize}
+                disk_type = overrides.get('disk_type', None)
```

Comment:
we don't want to use a global setting, instead parse the current disk item so you would rather do 

```
boot_disk_type = disk.get('interface')
if boot_disk_type is not None:
   ....
```

I use the keyword interface here as it's one that already exists